src/Transition.js

```diff
@@ -590,3 +575,17 @@ Transition.ENTERED = ENTERED
 Transition.EXITING = EXITING
 
 export default Transition
+
+function getTimeouts(timeout) {
```

Comment:
**nitpick:** we should generally define methods before we use them, so maybe lift this to the top?

Reply:
Should we take into consideration that functions are hoisted to the top automatically?

Reply:
right this is purely a code style thing